### PR TITLE
Add Nap Option to Sandwich.md

### PR DESCRIPTION
**Pull request by proticom**
Please review this, @casey-collab

### CODE REVIEW

File: english/take-nap/cucumber-induced-nap.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+You unpack your porta-bed that you got from [Costco](http://www.costco.com) 
+And lay your head on the [MyPillow](http://www.mypillow.com) that you bought
```
Comment (casey-collab) [inline english/take-nap/cucumber-induced-nap.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/take-nap/cucumber-induced-nap.md (hunk 1 above)
Comment (proticom) [inline english/take-nap/cucumber-induced-nap.md]: These are not options @casey-collab

Comment (casey-collab) [inline english/take-nap/cucumber-induced-nap.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by proticom** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by proticom** [comment]
How on earth do I respond.

**Comment by proticom** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**